modules/ROOT/partials/configuration/toolbar_mode.adoc

```diff
@@ -63,12 +66,14 @@ tinymce.init({
 });
 ----
 
+[[sliding]]
 ==== Sliding
 
-If the `toolbar_mode` is configured to `sliding`, the toolbar appears as a fixed toolbar under the first toolbar when the toolbar overflow icon image:{baseurl}/images/icons/more-drawer.svg[Drawer] is clicked.
+If the `toolbar_mode` is configured to `sliding`, the toolbar appears as a fixed toolbar under the first toolbar when the toolbar overflow icon image:icons/more-drawer.svg[Drawer] is clicked.
 
-image::{baseurl}/images/sliding_toolbar.png[Sliding Toolbar]
+image:sliding_toolbar.png[Sliding Toolbar]
```

Comment:
This change is incorrect as it's meant to be a block image, not an inline image.
```suggestion
image::sliding_toolbar.png[Sliding Toolbar]
```